### ISSUE TITLE
Board design plus modified rule set

### ISSUE THREAD

**Issue by BrickMonkey** [dylan/gameofcyvasse#14]
Hi Dylan. I love the simplicity of your rule set and the idea of positionable terrain tiles. I've recently designed a LEGO Cyvasse board designed to implement this rule set: https://www.flickr.com/photos/37988267@N05/15546778970/. One point I found unclear was whether you meant for ranged weapons to be able to both move and fire on a single turn. After play testing I've found it preferable to disallow this.

I've also come up with a other few suggestions that seem to work well. I've strengthened the Rabble by allowng them to attack any unhorsed infantry (i.e. they can now attack crossbowmen), and I've strengthened elephants by making them invulnerable to the ranged weapons. I've also simplified water and forest tiles (while making them less of a barrier for ranged weapons) by having both simply reduce movement (cavalry) and firing range (ranged weapons) by one. The only difference between forest and water is now protection against dragons. And I've eliminated the need to track movement direction for the higher-ranked ranged weapon by having it fire only in diagonal or orthogonal lines (i.e. an asterisk-pattern instead of a square-zone).

UNIT RANKINGS: Rabble, Spearmen, Crossbowmen, Light Horse, Heavy Horse, Elephant, Scorpion, Catapult, Dragon, King. (Replaced trebuchet/catapult with catapult/scorpion simply due to LEGO build preference.)

RABBLE: May attack crossbowmen and below, i.e. all unhorsed infantry.
SPEARMEN: May attack crossbowmen and below, plus invulnerable to heavy & light horse.
CROSSBOWMEN: May attack light horse and below plus dragons. Firing range of 1 (-1 in forest or water).
LIGHT HORSE: May attack heavy horse and below except for spearmen. Movement of 3 (-1 in forest or water).
HEAVY HORSE: May attack elephants and below except for spearmen. Movement of 2 (-1 in forest or water).
ELEPHANT: May attack scorpion and below, plus invulnerable to ranged weapons.
SCORPION: May attack all but elephants. Firing range of 2 (-1 in forest or water).
CATAPULT: May attack all but elephants. Firing range of 3 but only in straight (i.e. diagonal or orthogonal) lines (-1 in forest or water).
DRAGON: May attack all units except those in forests. May land on mountains. Movement unlimited in straight lines, and blocked only by other dragon.
KING: May attack all units. Vulnerable to all units.

GRASS: No effect.
FOREST: Reduces firing range by 1 for ranged weapons. Reduces cavalry movement by 1 on entry and exit.
WATER: Same as forest, but without potection against dragons.
MOUNTAINS: Can be occupied only by dragons. Cannot be fired over or through. Barrier to all movement except dragons.
FORTRESS: Can be occupied by any unit from either side. Can be destroyed by any unit capable of defeating occupying unit.

**Comment by BrickMonkey** [comment]
I've since adjusted this further, giving both horse movement of 3, elephants movement of 2, and allowing units to attack those ranked 2 (rather than just 1) higher than themselves.

I also found it necessary to have the fortress reduce firing range for scorpions and catapults, as otherwise the catapult in a fortress on the frontlines became devastating and nearly unbeatable without sacrificing a dragon.

**Comment by dylan** [comment]
> One point I found unclear was whether you meant for ranged weapons to be able to both move and fire on a single turn.

Please see [the ranged section](http://gameofcyvasse.com/#ranged), and note that an attack ends the turn. If a ranged unit moves and then attacks they are probably over extending and risking loss rather than defending which is their ideal purpose.

> I've strengthened the Rabble by allowing them to attack any unhorsed infantry (i.e. they can now attack crossbowmen)

A rule that I have not typed up yet allows any unit to take a any ranged unit via attack, this should allow for significant mobility.

In regards to everything else you've added, it all has a compounding effect. You're welcome to play as you like (it's all about having fun after all) but each of these changes has a substantial impact on the whole and I'd like to address each issue independently to aid in a fair/dynamic and simple game.

Thanks for the feedback!

**Comment by BrickMonkey** [comment]
Cheers! I like the idea of making ranged weapons vulnerable to attack by more units. Though in practice I've found that few units can get close enough to hit them, which is why I've found it useful to bump up the movement range of the heavy horse and elephants. I'll keep play testing to see how all these suggested changes work out. I also tried playing on a 10x10 board for a while, but have switched back to 8x8. Of course, I don't expect you to adopt any of my suggestions. I'm just having fun playing with adjustments to the rule set and sharing what I've found!
